### ISSUE TITLE
grid_sampler_2d_backward error

### ISSUE THREAD

**Issue by naabdi** [facebookresearch/jacobian_regularizer#6]
Hello. I'm on Pytorch 2.0 and when use this regularizer, when running loss.backward(), I encounter with this error:
> derivative for aten::grid_sampler_2d_backward is not implemented
